source/backend/tests/unit/api/Services/LeaseServiceTest.cs

```diff
@@ -523,6 +684,42 @@ public void UpdateProperties_MatchProperties_Success()
             propertyService.Verify(x => x.UpdateLocation(It.IsAny<PimsProperty>(), ref It.Ref<PimsProperty>.IsAny, It.IsAny<IEnumerable<UserOverrideCode>>()), Times.Once);
         }
 
+        [Fact]
+        public void UpdateProperties_MatchProperties_Success_NoInternalId()
+        {
+            // Arrange
+            var lease = EntityHelper.CreateLease(1);
```

Comment:
I believe this test was already added in your other PR. Not sure if it would end up in a duplicate test when both PRs get merged